src/app/components/OptimoPromos/Lists/index.jsx

```diff
@@ -0,0 +1,21 @@
+import { GEL_SPACING_DBL } from '#app/legacy/gel-foundations/src/spacings';
+import styled from '@emotion/styled';
+
+// Bases
```

Comment:
The standards recommend not to add obvious comment noise

Reply:
Removed